src/sagemaker/amazon/common.py

```diff
@@ -16,7 +16,7 @@
 
 import numpy as np
 from scipy.sparse import issparse
-
+import json
```

Comment:
Please maintain the import order:

1.- python built in libraries
2.- 3rd party imports
3.- local library imports (from sagemaker...)